src/util/datablock/datablock_iterator.c

```diff
@@ -6,9 +6,16 @@
 */
 
 #include "datablock_iterator.h"
+#include "datablock.h"
 #include "assert.h"
 #include <stdio.h>
 
+int static inline _IsItemDeleted(int itemSize, char* item) {
+    char deleted = 0xFF;
+    for(int i = 0; i < itemSize; i++) deleted &= (item[i] & DELETED_MARKER);
+    return deleted;
+}
+
 DataBlockIterator *DataBlockIterator_New(Block *block, int start_pos, int end_pos, int step) {
```

Comment:
Should change start_pos and end_pos to be int64_t to match datablock.c:117

Reply:
👍